ale_linters/handlebars/embertemplatelint.vim

```diff
@@ -34,6 +34,6 @@ call ale#linter#Define('handlebars', {
 \   'executable': {b -> ale#node#FindExecutable(b, 'handlebars_embertemplatelint', [
 \       'node_modules/.bin/ember-template-lint',
 \   ])},
-\   'command': '%e --json %t',
+\   'command': '%e --json < %t',
```

Comment:
If you take `< %t` away from the command, ALE should pipe in stdin to the command automatically. ALE does that whenever `%t` isn't present in the command, and the `read_buffer` option isn't set to `0`.

Reply:
Ok, I didn’t know that. I will update accordingly.

Reply:
@dcyriller - We should also pass the current file name in via `--filename` (so that the linting rules that make changes based on filename have the correct value).